tests/AnalyzeSpec.hs

```diff
@@ -1600,7 +1654,10 @@ spec = describe "analyze" $ do
           |]
 
     expectVerified code
-    expectPass code $ Valid Success'
+    expectPass code $ Valid $
+      PropSpecific (RowExists "ints" "any index" Before)
+      <=>
+      Success'
```

Comment:
This is great to have all of these now

Reply:
It was cool to see the system start complaining when I turned on row existence checking.